doc/iterations/2023.6/scopes/addPagesToForm.adoc

```diff
@@ -0,0 +1,11 @@
+= Scope
+
+== First task
+
+- [ ] Shape written, reviewed and merged
+- [ ] ADR written, reviewed and merged
+- [ ] View DSL updated
+- [ ] Mock of the features done with Figma
+- [ ] First prototype of the user interface
+- [ ] First version of the user interface done with Cypress tests
+- [ ] Capture proper screenshots for the CHANGELOG and the documentation
```

Comment:
Maintain the scope file on your side and contribute it when the work is done. Before the iteration, we are only working on shapes, not scopes (which should be maintained / done when the work is really being done). Some shapes will be written for tasks which will only start in a couple weeks and things may change by then.

No need to mention the issue in the commit / pr, just use something like `[doc] Contribute a shape for the addition of pages to the view DSL`. Some shapes may end up creating multiple issues and you would have to update the changelog just for that.

Reply:
I removed the scope and file and changed the commit message.